### ISSUE TITLE
validate the initial condition data

### ISSUE THREAD

**Issue by nbren12** [VulcanClimateModeling/fv3config#22]
There are some simple checks we could perform on the initial condition data to make sure it is compatible with the fv3gfs. Here are a couple of examples:

- The restart files need to only contain `float` values, otherwise the model errors out.
- The grid needs to match the one specified in the `input.nml`. 

Ultimately, this feature should probably be part of a bigger refactor of the representation of the initial condition data to a richer python object.

**Comment by mcgibbon** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [FV3Config: Validate initial conditions data](https://trello.com/c/SP3k0lnk/127-fv3config-validate-initial-conditions-data)

**Comment by oliverwm1** [comment]
Agreed some checking on initial condition data would be useful. This is one aspect of what I suggested in #6 although we got a little sidetracked in that issue.
